src/IW/App.hs

```diff
@@ -0,0 +1,9 @@
+module IW.App
+( module IW.App.Env
+, module IW.App.Error
```

Comment:
According to our style guide, `(` and `,` should be aligned under module name with 7 spaces. Like this:

```haskell
module IW.App
       ( module IW.App.Env
       , ...
       ) where
```

Reply:
Yep that's wrong. Will fix it. Thank you.